libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.h

```diff
@@ -115,6 +115,19 @@ typedef void (^SFSnapshotViewControllerDismissalBlock)(UIViewController* snapsho
 + (nonnull NSString *)ailtnAppName;
 
 /**
+   Gets an instance of WKProcessPool that will be used during instantiation of any WKWebView instances
+  @discussion
+  Use the default instance to share state between WKWebView instances
+  */
++ (nonnull WKProcessPool *) processPool;
```

Comment:
Given that we've flipped out of dependency on `SalesforceSDKManager` for the process pool, I would very much like to do away with the dependency in the SDK manager, and simply access the process pool directly from the consumers.

`SalesforceSDKManager` tends to carry some baggage with it when it's included around in other classes, arguably creating circular dependencies in others.  Since we have a standalone facility in this implementation, let's use it directly.